FCCee/ALLEGRO/compact/ALLEGRO_o1_v03/ALLEGRO_o1_v03.xml

```diff
@@ -25,21 +25,23 @@
     <constant name="world_size" value="25*m"/>
     <constant name="world_x" value="world_size"/>
     <constant name="world_y" value="world_size"/>
-    <constant name="world_z" value="world_size"/>
+    <constant name="world_z" value="world_size"/>    
   </define>
 
   <include ref="./DectDimensions.xml" />
 
-  <include ref="Beampipe.xml"/>
+  <!--<include ref="Beampipe.xml"/>-->
+  <include ref="../../../MDI/compact/MDI_o1_v01/Beampipe_CADimport_o1_v02.xml" />
   <include ref="BeamInstrumentation.xml"/>
   <include ref="LumiCal.xml"/>
-  <include ref="HOMAbsorber.xml"/>
+  <!--<include ref="HOMAbsorber.xml"/>-->
```

Comment:
Can you comment on HOM Absorber? Needed, not needed, not known?

Reply:
HOM Absorbers are currently not needed anymore due to the low impedance IR beam pipe. Some smaller HOM-A will be placed inside the bellows to absorb the modes produced by those elements, and will be added to the model together with the bellows once ready.